### PR TITLE
Swap to new eslint-config-condenast

### PR DESCRIPTION
**Pull request by tswicegood**

### CODE REVIEW

**Comment by tswicegood** [comment]
[This gist](https://gist.github.com/anonymous/cb5ecd70f9e471658187af1a48b96a47) has the output before any code changes. If you want to be able to use this without making code changes, we could add these as overrides to the `user-platform` team. I'll leave that up to y'all.

File: .eslintrc [hunk 1]
```diff
@@ -1,15 +0,0 @@
...
-  "env": {
-    "node": true,
-    "mocha": true,
-    "es6": true
-  },
-  "rules": {
-    "strict": [0, "global"],
-    "global-strict": [0, "always"],
-    "indent": [1, 2],
-    "consistent-return": 0,
```
Comment (pgoldrbx) [inline .eslintrc]: interesting that this rule was turned off — consistent return is important to allow for some potential V8 optimization

File: cluster.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ const log = require('./logger');
 
 const workers = [];
 let restart = true;
+var child;
```
Comment (pgoldrbx) [inline cluster.js]: use `let`?

Comment (pgoldrbx) [inline cluster.js]: actually - this isn't used anywhere except the scope below. why not just use `let` below inside the loop?

**Comment by pgoldrbx** [comment]
I had a couple minor questions/notes but overall seems like a pretty straightforward change

File: cluster.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ const log = require('./logger');
 
 const workers = [];
 let restart = true;
+let child;
```
Comment (ghvaldez) [inline cluster.js]: using let here as a use-before-defined error would be thrown if defined in scope below.

**Pull request merged**